RCECHal.pc.in

```diff
@@ -0,0 +1,10 @@
+prefix=@CMAKE_INSTALL_PREFIX@

```

Comment:
what this file used for, you need to add comments at the top..

Reply:
and why is it called `RCEHal.pc.in` - extension wise, and where is `RCEHal.pc`?

Reply:
CMake can create package config file using template (https://cmake.org/cmake/help/v3.1/command/configure_file.html?highlight=configure_file). RCEHal.pc.in is the template and RCEHal.pc is the final file which will be installed in /usr/share/pkgconfig/. If any third-party code need to find the RCECHal.so; it can use https://cmake.org/cmake/help/v3.1/command/find_package.html?highlight=find_package method to get the data exported by RCEHal.pc.